docs/root/configuration/http/http_conn_man/headers.rst

```diff
@@ -363,6 +363,21 @@ A few very important notes about XFF:
      XFF is parsed to determine if a request is internal. In this scenario, do not forward XFF and
      allow Envoy to generate a new one with a single internal origin IP.
 
+.. _config_http_conn_man_headers_x-forwarded-host:
+
+x-forwarded-host
+----------------
+
+The *x-forwarded-host* header is a standard proxy header which indicates the original host
```

Comment:
Nit: Technically there's no standard for this header. The [standard](https://www.rfc-editor.org/rfc/rfc7239#section-4) specifies `Forwarded` instead.
```suggestion
The *x-forwarded-host* header is a de-facto standard proxy header which indicates the original host
```

Reply:
Thank you, @rojkov. I just fixed that.